components/Card.vue

```diff
@@ -120,6 +120,29 @@ export default Vue.extend({
   methods: {
     imageLink(id: string) {
       return `${this.$axios.defaults.baseURL}/Items/${id}/Images/Primary`;
+    },
+    /**
+     * @returns {string} Empty sting || item Production year || item Production year - Present || item Production Year - item End Year
```

Comment:
I'm not too sure about this kind of comments...

It's a bit messy, imo. We don't really care what format of string it's returning, more what that string represents, I think.

Could be:
```suggestion
     * @returns {string} Either an empty string, or a string representing the production year for the current item.
```

Reply:
Sounds good I'll change it